### ISSUE TITLE
[Accessibility]: add two-dimensional navigation the CodeMirror blob view

### ISSUE THREAD

**Issue by taras-yemets** [sourcegraph/sourcegraph#44594]
### Audit type

Screen reader navigation

### Problem description

Follow-up on https://github.com/sourcegraph/sourcegraph/pull/44148.
Currently in the CodeMirror blob view a screen reader user can navigate blame and code content line by line. There's no possibility to go through only blob column or only code column line by line. For more details please check out [this comment](https://github.com/sourcegraph/sourcegraph/pull/44148#issuecomment-1311160200).

### Expected behavior

A screen reader user should be able to navigate the code content in two dimensions: columns (blame or code) and rows (blame and code combined). For more details please check out [this comment](https://github.com/sourcegraph/sourcegraph/pull/44148#issuecomment-1311160200).

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
